index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
```

Comment:
Yes, a screen reader treats things like this:

- `alt="This is a man in a bear suit"`   "Image: This is a man in a bear suit"
- `alt not present` "Image: img-0239023403.jpg"
- `alt=""` "" <- total silence, it ignores that the image is there